File: 1078-remove-outermost-parentheses/1078-remove-outermost-parentheses.cpp
class Solution {
public:
    string removeOuterParentheses(string s) {
        string ass="";
        int count=0;
        for(char c:s) {
            if(c=='(') {
                if(count>0) ass.push_back(c);
                count++;
            }else{
                count--;
                if(count>0) ass.push_back(c);
            }
        }
        return ass;
    }
};

File: 1078-remove-outermost-parentheses/1078-remove-outermost-parentheses.py
class Solution:
    def removeOuterParentheses(self, s: str) -> str:
        ass=[]
        count=0
        for c in s:
            if c=="(":
                if count>0: ass.append(c)
                count+=1
            else:
                count-=1
                if count>0: ass.append(c)
        return "".join(ass)